Eshop.WebApi/Controllers/CategoryController.cs

```diff
@@ -7,7 +8,14 @@ namespace Eshop.WebApi.Controllers
     [Route("[controller]")]
     public class CategoryController : Controller
     {
-        public static List<Category> Categories = new List<Category>
+        private readonly EshopDbContext DbContext;
+
+        public CategoryController(EshopDbContext dbContext)
+        {
+            DbContext = dbContext;
+        }
+
+        public static List<Category> categories = new List<Category>
```

Comment:
your project is not buildable - 
![image](https://github.com/user-attachments/assets/cc55ebf0-b19b-4e70-a95b-94e27d6341a4)

Reply:
fixed